data/pathfinder/paizo/roleplaying_game/mythic_adventures/ma_equipmods.lst

```diff
@@ -28,7 +28,7 @@ Intelligent			KEY:LEGEND_INTELLIGENT		TYPE:Legendary	VISIBLE:QUALIFY	ITYPE:Mythi
 Legendary Fortification	KEY:LEGENDARY_FORTIFICATION	TYPE:Legendary	VISIBLE:QUALIFY					SOURCEPAGE:p.172	PRETYPE:1,Armor,Shield,WornItem,Ring,Amulet,Cape,Headgear		SPROP:the wearer can expend one use of legendary power to negate the critical hit or sneak attack and instead take normal damage.
 Metamagician		KEY:LEGEND_METAMAGICIAN		TYPE:Legendary	VISIBLE:QUALIFY					SOURCEPAGE:p.173	PRETYPE:1,HeadSlot,HeadbandSlot,Headgear,Headband,Staff,Ring,Rod	SPROP:allows a spellcaster to apply a metamagic feat she knows to a spell as she casts it. She must expend a number of uses of legendary power equal to the increase of spell level the metamagic feat usually applies (minimum 1). This metamagic feat is applied spontaneously and without changing the casting time.
 Perfect Surge		KEY:LEGEND_PERFECT_SURGE	TYPE:Legendary	VISIBLE:QUALIFY					SOURCEPAGE:p.173	PRETYPE:1,MajorArtifact								SPROP:item's legendary surge can apply to any d20 roll. The surge adds a further +2 bonus when applied to one of the types of rolls initially chosen for it.
-Powerful			KEY:LEGEND_POWERFUL		TYPE:Legendary	VISIBLE:QUALIFY					SOURCEPAGE:p.173	PRETYPE:1,MinorArtifact,MajorArtifact					SPROP:item with this ability has two additional uses of legendary power per day. This ability can be taken up to three times.
+Powerful			KEY:LEGEND_POWERFUL		TYPE:Legendary	VISIBLE:QUALIFY					SOURCEPAGE:p.173															SPROP:item with this ability has two additional uses of legendary power per day. This ability can be taken up to three times.
```

Comment:
I'm not seeing the issue with the way this is currently implemented, from what I read on TYPE/PRETYPE/ITYPE at least.
TYPE should a list of item types that have the ability to use this equipment mod, in this case, any item that has (and qualifies for) Legendary.
PRETYPE should be prereqs that enabled this mod, in this case Nothing,MinorArtifact,MajorArtifact.
ITYPE only applies to Intelligent because it's the only one that unlocks further sub-abilities.